docs/DeveloperGuide.md

```diff
@@ -562,16 +564,17 @@ The sequence of the `logout` command is as follows:
 
 1. The user inputs the `logout` command.
 2. A new `userLogoutCommand` is created and checks whether the user is currently logged out
-   by calling `Model#getUserLoginStatus()`.
-3. If the user is currently logged in, the `userLogoutCommand` calls `Model#setLogoutSuccess()`,
+   by calling `Model#getUserLoginStatus`.
+3. If the user is currently logged in, the `userLogoutCommand` calls `Model#setLogoutSuccess`,
    changing the login status to false and restricting the user access to most commands.
-4. The `userLoginCommand` also calls `Model#updateFilteredPersonList()` to hide the list of customers.
+4. The `userLoginCommand` also calls `Model#clearFilteredDeliveryList` and `Model#clearFilteredCustomerList`
```

Comment:
Is this supposed to be userLogoutCommand?

Reply:
Good catch, thanks!